### PR TITLE
Added the Terraform Create Variable and Apply Run Cloudslang Operations

### PR DESCRIPTION
**Pull request by KalyankarRajesh**
Added the Terraform Create Variable and Apply Run Cloudslang Operations.

Added the Junit tests for Create Variable and Apply Run

### CODE REVIEW

File: cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/runs/ApplyRun.java [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+                    @Output(value = STATUS_CODE, description = STATUS_CODE_DESC)
+            },
+            responses = {
+                    @Response(text = ResponseNames.SUCCESS, field = OutputNames.RETURN_CODE, value = ReturnCodes.SUCCESS, matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.RESOLVED, description = SUCCESS_DESC),
+                    @Response(text = ResponseNames.FAILURE, field = OutputNames.RETURN_CODE, value = ReturnCodes.FAILURE, matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.ERROR, description = FAILURE_DESC)
+            })
+    public Map<String, String> execute(@Param(value = AUTH_TOKEN, required = true, encrypted = true, description = AUTH_TOKEN_DESC) String authToken,
+                                       @Param(value = RUN_ID, required = true, description = RUN_DESC) String runId,
+                                       @Param(value = RUN_COMMENT, description = RUN_COMMENT_DESC) String runComment,
+                                       @Param(value = REQUEST_BODY) String requestBody,
```
Comment (IuliaMoldovan) [inline cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/runs/ApplyRun.java]: does not have description

Comment (KalyankarRajesh) [reply cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/runs/ApplyRun.java]: incorporated the review comments

File: cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/runs/ApplyRun.java [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+                connectTimeout, socketTimeout, keepAlive, connectionsMaxPerRoute, connectionsMaxTotal);
+        if (!exceptionMessage.isEmpty()) {
+            return getFailureResultsMap(StringUtilities.join(exceptionMessage, NEW_LINE));
+        }
+
+        final List<String> exceptionMessages = verifyApplyRunRequestBody(requestBody);
+        if (!exceptionMessages.isEmpty()) {
+            return getFailureResultsMap(StringUtilities.join(exceptionMessages, NEW_LINE));
+        }
+
```
Comment (ericandreica) [inline cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/runs/ApplyRun.java]: You can use a bigger function that include both verification so you won't need to create two lists.

File: cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/variables/CreateVariable.java [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+                                       @Param(value = TRUST_ALL_ROOTS, description = TRUST_ALL_ROOTS_DESC) String trustAllRoots,
+                                       @Param(value = X509_HOSTNAME_VERIFIER, description = X509_DESC) String x509HostnameVerifier,
+                                       @Param(value = TRUST_KEYSTORE, description = TRUST_KEYSTORE_DESC) String trustKeystore,
+                                       @Param(value = TRUST_PASSWORD, encrypted = true, description = TRUST_PASSWORD_DESC) String trustPassword,
+                                       @Param(value = CONNECT_TIMEOUT, description = CONNECT_TIMEOUT_DESC) String connectTimeout,
+                                       @Param(value = SOCKET_TIMEOUT, description = SOCKET_TIMEOUT_DESC) String socketTimeout,
+                                       @Param(value = EXECUTION_TIMEOUT, description = EXECUTION_TIMEOUT_DESC) String executionTimeout,
+                                       @Param(value = ASYNC, description = ASYN_DESC) String asyn,
+                                       @Param(value = POLLING_INTERVAL, description = POLLING_INTERVAL_DESC) String pollingInterval,
```
Comment (ericandreica) [inline cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/variables/CreateVariable.java]: You should use async not asyn all over the places.

File: cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/variables/CreateVariable.java [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+        final List<String> exceptionMessage = verifyCommonInputs(proxyPort, trustAllRoots,
+                connectTimeout, socketTimeout, keepAlive, connectionsMaxPerRoute, connectionsMaxTotal);
+        if (!exceptionMessage.isEmpty()) {
+            return getFailureResultsMap(StringUtilities.join(exceptionMessage, NEW_LINE));
+        }
+        final List<String> exceptionMessages = verifyCreateVariableInputs(workspaceId, variableName, variableValue, variableCategory, requestBody);
+        if (!exceptionMessages.isEmpty()) {
+            return getFailureResultsMap(StringUtilities.join(exceptionMessages, NEW_LINE));
+        }
+
```
Comment (ericandreica) [inline cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/variables/CreateVariable.java]: You can use a bigger function to verify more inputs.

**Pull request merged**